core/src/main/java/org/geysermc/geyser/item/type/ArmorItem.java

```diff
@@ -33,6 +33,7 @@
 import org.geysermc.geyser.session.GeyserSession;
 
 public class ArmorItem extends Item {
+    private static final String MINECRAFT_NAMESPACE = "minecraft:";
```

Comment:
You can just use the string inline as that seems to be what we've done in the rest of the codebase.